### PR TITLE
Updated Czech translation and a few fixes

### PR DESCRIPTION
**Pull request by mipa87**
Helllo,
I've updated the Czech translation and made a few minor corrections

### CODE REVIEW

File: Slim/Plugin/MusicMagic/strings.txt [hunk 1]
```diff
@@ -123,7 +123,7 @@ SETUP_MIX_FILTER
 	ES	Filtro de mezcla
 	FI	Miksin suodatin
 	FR	Filtre de mix
-	HE	מסנן מיקסים של 
+	HE	מסנן מיקסים של
```
Comment (mherger) [inline Slim/Plugin/MusicMagic/strings.txt]: Could you please revert this change?

File: Slim/Plugin/Podcast/strings.txt [hunk 2]
```diff
@@ -252,51 +259,61 @@ PLUGIN_PODCAST_SKIP_BACK
...
 	DE	Neue Episoden der letzten %s Tage
 	EN	New episodes of the last %s days
 	FR	Nouveaux épisodes des %s derniers jours
 
 PLUGIN_PODCAST_AUTHOR
+	DE	Autor:
 	DE	Herausgeber:
 	EN	Author:
-	FR	Auteur :
+	FR	Auteur:
```
Comment (mherger) [inline Slim/Plugin/Podcast/strings.txt]: Please revert this change: French does put a space before the colon.

File: Slim/Plugin/RemoteLibrary/strings.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 PLUGIN_REMOTE_LIBRARY_MODULE_NAME
+	DE	Vzdálené hudební knihovny
```
Comment (mherger) [inline Slim/Plugin/RemoteLibrary/strings.txt]: This should be `CS` 😉

File: strings.txt [hunk 4]
```diff
@@ -6894,9 +6895,9 @@ SETUP_REFRESHRATE_DESC
...
-	EN	Set the time between browser reloads in seconds.  While playing a song, Logitech Media Server will set the refresh near a song end if it is less than this value. (Default is 30, minimum is 2)
+	EN	Set the time between browser reloads in seconds. While playing a song, Logitech Media Server will set the refresh near a song end if it is less than this value. (Default is 30, minimum is 2)
 	ES	Establezca en segundos el tiempo de espera entre las recargas del navegador. Mientras se reproduce una canción, Logitech Media Server establecerá la actualización cerca del final de una canción si es inferior a este valor. (Valor predeterminado: 30, mínimo: 2)
-	FI	Määritä selaimen päivitysten välillä kuluva aika sekunteina.  Kappaleen toiston aikana Logitech Media Server päivittää selaimen lähellä kappaleen loppua, jos jäljellä oleva aika on lyhyempi kuin tämä arvo (Oletus on 30, vähimmäisaika 2).
+	FI	Määritä selaimen päivitysten välillä kuluva aika sekunteina. Kappaleen toiston aikana Logitech Media Server päivittää selaimen lähellä kappaleen loppua, jos jäljellä oleva aika on lyhyempi kuin tämä arvo (Oletus on 30, vähimmäisaika 2).
```
Comment (mherger) [inline strings.txt]: Please revert.

**Review by mherger — CHANGES_REQUESTED** [review]
Thank you _very_ much! This is highly appreciated.

May I ask you to revert any change to languages other than Czech? Not even whitespace? Some languages are picky about where to put white space, and certainly different from other languages. Thanks!

**Comment by mipa87** [comment]
Updated and fixed 🙂

**Comment by mherger** [comment]
Thank you very much!
